lib/js_of_ocaml/resizeObserver.ml

```diff
@@ -0,0 +1,80 @@
+(* Js_of_ocaml library
+ * http://www.ocsigen.org/js_of_ocaml/
+ * Copyright (C) 2019 Alexander Yanin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, with linking exception;
+ * either version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ *)
+
+class type resizeObserverSize =
+  object
+    method inlineSize : float Js.readonly_prop
+
+    method blockSize : float Js.readonly_prop
+  end
+
+class type resizeObserverEntry =
+  object
+    method target : Dom.node Js.t Js.readonly_prop
+
+    method contentRect : Dom_html.clientRect Js.t Js.readonly_prop
+
+    method borderBoxSize : resizeObserverSize Js.t Js.readonly_prop
+
+    method contentBoxSize : resizeObserverSize Js.t Js.readonly_prop
+  end
+
+class type resizeObserverOptions =
+  object
+    method box : Js.js_string Js.t Js.prop
```

Comment:
Should this be `Js.writeonly_prop` so that the following is disallowed ?
```ocaml
let o = empty_resize_observer_options ()
o##.box
```

Reply:
I tend to agree, especially considering the similar implementation for mutationObserver bindings. Didn't notice that before. Should I do a commit?

Reply:
Yes please.

Reply:
Done